<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="CDC2-D-519QS77" ignored="0" name="Assertion" tests="3" failures="1" timestamp="12 Nov 2019 11:02:39 GMT" time="0.142" errors="0">
  <testcase name="test" time="0.027" classname="TestDemo_Assertion.TestDemo_Assert"/>
  <testcase name="test1" time="0.006" classname="TestDemo_Assertion.TestDemo_Assert"/>
  <testcase name="test2" time="0.005" classname="TestDemo_Assertion.TestDemo_Assert">
    <failure type="java.lang.AssertionError" message="expected [120] but found [20]">
      <![CDATA[java.lang.AssertionError: expected [120] but found [20]
at TestDemo_Assertion.TestDemo_Assert.test2(TestDemo_Assert.java:25)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- test2 -->
</testsuite> <!-- Assertion -->
